cns/restserver/ipam.go

```diff
@@ -88,8 +88,8 @@ func (service *HTTPRestService) requestIPConfigHandler(w http.ResponseWriter, r
 		return
 	}
 
-	// This method can only return 1 IP. If we have more than one NC then we expect to need to return one IP per NC
-	if len(service.state.ContainerStatus) > 1 {
+	// This method can only return EXACTLY 1 IP. If we have more than one NC then we expect to need to return one IP per NC
+	if len(service.state.ContainerStatus) != 1 {
```

Comment:
this will now work when the len is 0. is that intended?

Reply:
This is an error check. Before it was looking for when we have more than 1 NC since it will try to allocate 1 IP per NC. Now it will also hit this error when we have 0 NCs